system/COPY/etc/motd

```diff
@@ -0,0 +1,13 @@
+Welcome to the
+                   _ _
+  __ _ _ __  _ __ | (_) __ _ _ __   ___ ___
+ / _` | '_ \| '_ \| | |/ _` | '_ \ / __/ _ \
+| (_| | |_) | |_) | | | (_| | | | | (_|  __/
+ \__,_| .__/| .__/|_|_|\__,_|_| |_|\___\___|
+      |_|   |_|            _
+  ___ ___  _ __  ___  ___ | | ___
+ / __/ _ \| '_ \/ __|/ _ \| |/ _ \
+| (_| (_) | | | \__ \ (_) | |  __/
+ \___\___/|_| |_|___/\___/|_|\___|
```

Comment:
The command `appliance_console` is consistent across languages.
So only the "Welcome to" and "For a menu, please type:" would need to change.

We could just do nothing and let them read the docs to know to type "appliance_console".
Or we could just delete the `appliance_console` and use `cloud-init` instead.